openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -1024,10 +1024,16 @@
         OpenShift uses various TLS algorithms and ciphers to protect
         the authenticity of communications sessions.
 
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        HTTP Strict Transport Security (HSTS) policy is a security
+        enhancement, which ensures that only HTTPS traffic is
+        allowed on the host. Any HTTP requests are dropped by default.
+        This is useful for ensuring secure interactions with websites,
+        or to offer a secure application for the user’s benefit.
+
+        https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
```

Comment:
In addition to HSTS, are there other settings that may impact the authenticity of communication sessions? For example:
- Server side PKI certs
- User PKI